frontend/packages/ceph-storage-plugin/src/components/odf-system/utils.ts

```diff
@@ -9,41 +19,57 @@ export const getGVK = (label: string) => {
   return { kind, apiGroup, apiVersion };
 };
 
-export type SystemMetrics = {
-  metrics: {
-    [systeName: string]: {
-      rawCapacity: HumanizeResult;
-      usedCapacity: HumanizeResult;
-      iops: HumanizeResult;
-      throughput: HumanizeResult;
-      latency: HumanizeResult;
-    };
+type SystemMetrics = {
+  [systeName: string]: {
+    rawCapacity: HumanizeResult;
+    usedCapacity: HumanizeResult;
+    iops: HumanizeResult;
+    throughput: HumanizeResult;
+    latency: HumanizeResult;
   };
 };
 
 type MetricNormalize = (
+  systems: StorageSystemKind[],
   latency: PrometheusResponse,
   throughput: PrometheusResponse,
   rawCapacity: PrometheusResponse,
   usedCapacity: PrometheusResponse,
   iops: PrometheusResponse,
-) => SystemMetrics['metrics'];
+) => SystemMetrics;
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 export const normalizeMetrics: MetricNormalize = (
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  _latency,
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  _throughput,
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  _rawCapacity,
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  _usedCapacity,
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  _iops,
+  systems,
+  latency,
+  throughput,
+  rawCapacity,
+  usedCapacity,
+  iops,
 ) => {
-  // Todo(bipuladh): Add parsing logic for above items
-  return {
-    metrics: {} as any,
-  };
+  if (_.isEmpty(systems) || !latency || !throughput || !rawCapacity || !usedCapacity || !iops) {
+    return {};
+  }
+  return systems.reduce<SystemMetrics>((acc, curr) => {
```

Comment:
nits(if it seems alright): return systems.reduce<SystemMetrics>((metrics, system) => {

Reply:
Since the initial value supplied at line 74 is an `object` the type for `acc` also becomes `{}` so to fix that I am supplying the type for initial value.

Reply:
I meant variable naming:
acc --> metrics
curr --> system